### ISSUE TITLE
It was working elegantly without any GTK!

### ISSUE THREAD

**Issue by ghost** [trizen/fbmenugen#1]
Not an issue, but is that recently added GTK dependency crucial for the otherwise very elegant fbmenugen? (Asking, because I don't have any GTK in my [Arch] system.)

**Comment by trizen** [comment]
I think we can make it optional, as in generating a menu without icons, the GTK library is not really needed. 

This decision was made because the old system which was used for resolving the absolute icon paths for icons, was not very robust and with known issues in several special cases. The new system, which uses GTK, is both more faster, more robust and also added support for SVG icons, which were not supported before.

In conclusion, if you are not using icons inside the menu, you should still be able to use fbmenugen without having the GTK library installed. I will make this dependency optional in the AUR package as well. Thanks for reporting this.

Edit: the AUR package now lists `gtk2-perl` as optional.

**Comment by ghost** [comment]
Thanks a lot. I'm aware of
> the old system which was used for resolving the absolute icon paths for icons, was not very robust

yet, it's better to have it optional as I can keep my system lean without GTK. Thanks as well for a very useful utility, fbmenugen.

**Comment by trizen** [comment]
You're very welcome. I'm glad to know you find `fbmenugen` useful.

The [Gtk2](https://metacpan.org/pod/Gtk2) dependency is now fully optional when a menu without icons is generated. 

Sorry for the inconvenience caused.
